variables.tf

```diff
@@ -36,12 +36,48 @@ variable "data_egress_server_ebs_volume_type" {
     production  = "gp2"
   }
 }
-variable "al2_hardened_ami_id" {
-  description = "The AMI ID of the latest/pinned Hardened AMI AL2 Image"
+variable "ecs_hardened_ami_id" {
+  description = "The AMI ID of the latest/pinned ECS Hardened AMI Image"
   type        = string
 }
 variable "truststore_aliases" {
   description = "comma seperated truststore aliases"
   type        = list(string)
   default     = ["dataworks_root_ca", "dataworks_mgt_root_ca"]
 }
+variable "fargate_cpu" {
+  default = "512"
+}
+
+variable "fargate_memory" {
+  default = "512"
+}
+
+variable "receiver_cpu" {
+  default = "512"
+}
+
+variable "receiver_memory" {
+  default = "1024"
+}
+
+variable "data_egress_port" {
+}
+
+variable "image_versions" {
```

Comment:
I don't see a corresponding variable being set in the pipeline for this. We should be tracking a SHA